src/analysis/powerset-lattice-impl.h

```diff
@@ -5,61 +5,115 @@
 
 namespace wasm::analysis {
 
-template<size_t N>
-inline BitsetPowersetLattice<N> BitsetPowersetLattice<N>::getBottom() {
+inline FinitePowersetLattice::FinitePowersetLattice(size_t size) : trues(0) {
+  bitvector.reserve(size);
+  for (size_t i = 0; i < size; ++i) {
+    bitvector.push_back(false);
+  }
+}
```

Comment:
This can be made a one-liner, so it can probably just go in the .h file rather than the separate impl file.

```suggestion
inline FinitePowersetLattice::FinitePowersetLattice(size_t size) : bitvector(size), trues(0) {}
```